README.md

```diff
@@ -1,21 +1,83 @@
-[![GitHub release](https://img.shields.io/github/release/cybozu-go/neco-template.svg?maxAge=60)][releases]
-[![CircleCI](https://circleci.com/gh/cybozu-go/neco-template.svg?style=svg)](https://circleci.com/gh/cybozu-go/neco-template)
-[![PkgGoDev](https://pkg.go.dev/badge/github.com/cybozu-go/neco-template?tab=overview)](https://pkg.go.dev/github.com/cybozu-go/neco-template?tab=overview)
-[![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/neco-template)](https://goreportcard.com/report/github.com/cybozu-go/neco-template)
+[![GitHub release](https://img.shields.io/github/release/cybozu-go/pod-security-admission.svg?maxAge=60)][releases]
+[![CircleCI](https://circleci.com/gh/cybozu-go/pod-security-admission.svg?style=svg)](https://circleci.com/gh/cybozu-go/pod-security-admission)
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/cybozu-go/pod-security-admission?tab=overview)](https://pkg.go.dev/github.com/cybozu-go/pod-security-admission?tab=overview)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/pod-security-admission)](https://goreportcard.com/report/github.com/cybozu-go/pod-security-admission)
 
-Template repository for Neco
-============================
+Pod Security Admission
+======================
 
 **Project Status**: Initial development
 
+`pod-security-admission` is a Kubernetes [admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) to ensure [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
+
+The enforcement of security settings for Pods has previously been achieved using [Pod Security Policy (PSP)](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
+However, it was [announced](https://github.com/kubernetes/kubernetes/pull/97171) that PSP is going to be removed in Kubernetes 1.25.
+
+`pod-security-admission` provides an enforcement of [simple Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) as admission webhook.
+
+`pod-security-admission` is not a policy engine.
+It does not provide flexible policy changes.
+If you want to do that, I recommend using a policy engine such as [OPA GateKeeper](https://github.com/open-policy-agent/gatekeeper).
+
+Webhooks
+--------
+
+`pod-security-admission` provides 3 policy types based on Pod Security Standards.
+
+### Privileged
+
+The `Privileged` is an entirely unrestricted policy.
+Admission webhook will not work to Pods to which this policy is applied.
```

Comment:
Admission webhook does nothing to the Pods in namespaces with `Privileged` label.